src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkNotFoundException.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.appengine.cloudsdk;
+
+import com.google.cloud.tools.appengine.api.AppEngineException;
+
+/**
+ * User needs to use a valid Cloud SDK installation.
+ */
+public class CloudSdkNotFoundException extends AppEngineException {
```

Comment:
I'm not seeing a lot of value in the AppEngineException class of exceptions anymore.  It seems to be used by all exceptions that could possibly occur in the library rather than the unexpected errors caused by library deps.